main.tf

```diff
@@ -279,7 +279,7 @@ resource "google_container_node_pool" "node_pools" {
 }
 
 # https://cloud.google.com/iap/docs/using-tcp-forwarding
-resource "google_compute_firewall" "iap-tcp-forwarding" {
+resource "google_compute_firewall" "iap_tcp_forwarding" {
```

Comment:
if we claim any backward compatibility, we cannot do renames.

Reply:
I raised the same point, but was directed that the module was not used by customers. This will require a major version bump